include/experimental/__p1673_bits/blas1_scale.hpp

```diff
@@ -43,6 +43,8 @@
 #ifndef LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_BLAS1_SCALE_HPP_
 #define LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_BLAS1_SCALE_HPP_
 
+#include<execution>
```

Comment:
the current implementation (i.e. before my stuff) requires C++20 since it uses concepts unconditionally. I don't think we need to support C++14, 17 would be nice though.